### DISCUSSION TITLE
run_vivado卡住原因

### DISCUSSION THREAD

**Discussion by nthuyouwei** [bol-edu/HLS-SOC-Discussions#171] [Q&A] [answered]
想請問卡住原因
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145022311/b4bb3b03-f9b1-4cdb-a345-e02a0d3a3133)

![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145022311/f82476cb-3f9b-46d9-8c5d-261be01670c4)

tcl code 
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145022311/b44921b4-ac03-45e2-bf1a-0a84875af1ab)

![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145022311/7d127b35-adf1-468e-8ae1-5bd4a56c7970)

**Comment by huankai0731** [comment]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/94675784/8e5bda7a-ca08-4951-855b-e8e90d7d291e)
同學您好，如果你的code也是用原本提供的user_proj下去改的話，可以試試把這行的none改成wire

**Comment by nthuyouwei** [comment]
那已改過。還是一樣。

**Comment by Lanxas1018** [comment, marked as answer]
同學好
我當初tcl也有遇到同樣的問題，後來的解決方式是複製一行原先已打好的path，然後只更改新增的檔名部分就可以work了。
具體原因我不清楚，可能是vivado command line有些字元的要求

**Comment by nthuyouwei** [comment]
感謝，確實如上述同學的說明修改後就可以了
